tests/feature_elimination/test_feature_elimination.py

```diff
@@ -258,7 +258,7 @@ def test_shap_rfe_randomized_search_cols_to_keep(X, y, capsys):
     Test with ShapRFECV with column to keep param.
     """
     clf = DecisionTreeClassifier(max_depth=1)
-    param_grid = {"criterion": ["gini"], "min_samples_split": [1, 2]}
+    param_grid = {"criterion": ["gini"], "min_samples_split": [2, 4]}
```

Comment:
This bit puzzles me a bit why. I'll have to investigate a bit.

Reply:
I've ran your branch locally with Python version 3.8, 3.9, 3.10 and 3.11 where I've reverted this part back 

```
param_grid = {"criterion": ["gini"], "min_samples_split": [1, 2]}
```

I am not able to reproduce the error. Could you revert it back?

Reply:
Absolutely. I still see the error on my side after reverting. But will revert and push.